Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Button.cs

```diff
@@ -179,6 +187,19 @@ protected override void HandleUpdate()
             }
         }
 
+        /*
```

Comment:
Is this the chunk of code that's being left in but commented out to see if other people can figure out the button moving?